blocks/rich-text/index.js

```diff
@@ -212,6 +219,30 @@ export default class RichText extends Component {
 	}
 
 	onFocus() {
+		// For small screens (virtual keyboard), always scroll the focussed
+		// editor into view. Unfortunately we cannot detect virtual keyboards.
+		if ( window.innerWidth < 784 ) {
```

Comment:
MAybe we could extract this to a module const to avoid repetition

Reply:
Yeah, that's probably best. To utils?

Reply:
Hm, I just tried the iPad horizontally, and this won't cover it. We need a better way to detect virtual keyboards.